content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -115,6 +115,16 @@ The name of a higher-level application this object is part of.
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
+### kubernetes.io/bootstrapping
+
+Type: Label
+
+Example: ` kubernetes.io/bootstrapping=rbac-defaults`
+
+Used on: ClusterRoles and ClusterRoleBindings  
+
+The cluster roles and bindings can be viewed using this annotation.
```

Comment:
This seems misleading. What did you want to explain here?

Reply:
If you're not sure how to explain this label, it may be more appropriate to close the PR; this work is best done by someone who can explain how the label is used.